src/UI/WB.UI.Headquarters.Core/Controllers/Api/ExportSettingsApiController.cs

```diff
@@ -106,5 +153,26 @@ private async Task<bool> IsExistsDataExportInProgress()
         {
             return (await this.exportServiceApi.GetRunningExportJobs()).Any();
         }
+        
+        [HttpGet]
+        [ValidateAntiForgeryToken]
+        public async Task<IActionResult> StatusExportCache()

```

Comment:
Please rename to GetDropTenantStatus or DropTenantStatus.

Reply:
this name is used for ui request and it use 'export cache' meaning